src/lib/viewers/controls/annotations/annotations-button-region-tooltips/AnnotationsFlowTooltip.scss

```diff
@@ -1,14 +1,14 @@
-.bp-AnnotationsTooltip {
+.bp-AnnotationsFlowTooltip {
```

Comment:
It doesn't seem like these changes are strictly necessary. Can we revert?

Reply:
This connects to https://github.com/box/box-content-preview/pull/1445#discussion_r871310068 where I explained why I renamed this component.